### PR TITLE
Removing layer module

### PR DESCRIPTION
**Pull request by andreaswittig**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Removing the lamda-layer module as static exports do not work for layers.

### CODE REVIEW

**Comment by andreaswittig** [comment]
test

File: test/layer.yml [hunk 1]
```diff
@@ -8,17 +8,16 @@ Parameters:
...
+        S3Bucket: !Ref ContentBucket
+        S3Key: !Ref ContentKey
   Function:
     Type: 'AWS::CloudFormation::Stack'
     Properties:
       Parameters:
         Handler: 'defaults.handler'
         Runtime: 'nodejs8.10'
-        LayerModule: !GetAtt 'Layer.Outputs.StackName'
+        LayerArns: !Ref Layer
```
Comment (michaelwittig) [inline test/layer.yml:22]: Can you remove the lambda-layer dependency from test/package.json and run npm i again to update tets/package-lock.json as well?

File: README.md [hunk 2]
```diff
@@ -212,5 +205,19 @@ Resources:
...
+      <td>no</td>
+      <td></td>
+    </tr>
   </tbody>
 </table>
+
+
+## Migration Guides
+
+### Migrate to 2.x.x
```
Comment (michaelwittig) [inline README.md]: Migrate to v2

**Review by michaelwittig — CHANGES_REQUESTED** [review]

File: test/layer.yml (hunk 1 above)
Comment (andreaswittig) [reply test/layer.yml:22]: ✅

File: README.md (hunk 2 above)
Comment (andreaswittig) [reply README.md]: ✅

**Comment by andreaswittig** [comment]
test

**Review by michaelwittig — APPROVED** [review]

**Pull request merged**